src/app/(proper_react)/redesign/(authenticated)/user/(dashboard)/dashboard/View.tsx

```diff
@@ -130,8 +132,18 @@ export const View = (props: Props) => {
           onToggleExpanded={() => {
             if (exposureCardKey === activeExposureCardKey) {
               setActiveExposureCardKey(null);
+              recordTelemetry("collapse", "click", {
+                button_id: exposureCardKey
+                  .replaceAll("scan-", "data_broker_card_")
+                  .replaceAll("breach-", "data_breach_card_"),
```

Comment:
Since it's not obvious on line 125 that changing the format of `exposureCardKey` might break telemetry, maybe it would be safer to just construct the telemetry ID from scratch?

```suggestion
                button_id: isScanResult(exposure)
                  ? `data_broker_card_${exposure.onerep_scan_result_id}`
                  : `data_breach_card_${exposure.id}`
```